dz-03/corona.js

```diff
@@ -0,0 +1,60 @@
+const URL = "https://corona.lmao.ninja/countries";
+const keys = ["country", "cases", "deaths"];
+
+const main = () => {
+  const actions = () => {
+    const remove = document.createElement("button");
+    const removeText = document.createTextNode("Delete");
+    remove.appendChild(removeText);
+
+    remove.classList.add("btn");
+    remove.classList.add("red");
+
+    remove.addEventListener("click", function() {
+      this.parentElement.parentElement.remove();
+    });
+
+    return [remove];
+  };
+
+  fetch(URL)
+    .then(res => res.json())
```

Comment:
nedostaje provjera `res`, ako je status npr 404, vjeroajtno nece response biti Json i onda ce tu nastati greska kod poziva `res.json()`

Reply:
to ti je najlakse simulirati da namjerno odes na URL koji ne postoji